src/Components/Activities/index.js

```diff
@@ -1,9 +1,196 @@
+        method: 'POST',
+        body: JSON.stringify({
+          name: name,
+          description: description,
+          isActive: true
+        }),
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      });
+      const data = await res.json();
+      setActivities([...activities, data.data]);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  const onEdit = async (id) => {
+    const index = activities.findIndex((activity) => activity._id === id);
+    try {
+      const res = await fetch(`${process.env.REACT_APP_API}/api/activities/${id}`, {
+        method: 'PUT',
+        body: JSON.stringify({
+          name: currentName,
+          description: currentDes,
+          isActive: true
+        }),
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      });
+      const data = await res.json();
+      const update = [...activities];
+      update[index] = data.data;
+      console.log(update);
+      console.log(update[index]);
+      setActivities(update);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  const getActivities = async () => {
+    try {
+      const res = await fetch(`${process.env.REACT_APP_API}/api/activities/`);
+      const data = await res.json();
+      setActivities(data.data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  useEffect(() => {
+    getActivities();
+  }, []);
+
   return (
     <section className={styles.container}>
-      <h2>Activities</h2>
+      <div>
+        <h2>Activities</h2>
+        <div className={styles.tableCloth}>
+          {activities.map((activity) => {
+            return (
+              <table key={activity?._id}>
+                <tbody>
+                  <tr>
+                    <th className={styles.activity}>Activity</th>
+                    <th className={styles.description}>Description</th>
+                    <th>Edit</th>
+                    <th>Delete</th>
+                  </tr>
+                  <tr>
+                    <td className={styles.activity}>{activity?.name}</td>
+                    <td className={styles.description}>{activity?.description}</td>
+                    <td className={styles.icon}>
+                      <img
+                        src={editIcon}
+                        onClick={() => {
+                          setShowEdit(!showEdit);
+                          setShowAdd(false);
+                          setCurrentName(activity.name);
+                          setCurrentDes(activity.description);
+                          setCurrentId(activity._id);
+                        }}
+                      />
+                    </td>
+                    <td className={styles.icon}>
+                      <img
+                        src={deleteIcon}
+                        onClick={() => {
+                          if (confirm('Are you sure you want to delete?')) {
```

Comment:
I fixed everything but I dont know how to go back on the changes made to package.json and package-lock.json, I'm dragging those along since the first commit. Git ignore wont work on those files, I think because they were pushed allready.